examples/dump_torrent.cpp

```diff
@@ -100,7 +142,9 @@ int main(int argc, char* argv[]) try
 		std::printf("%2d: %s\n", i.tier, i.url.c_str());
 
 	std::stringstream ih;
-	ih << t.info_hash();
+	ih << t.info_hash().v1;
+	if (t.info_hash().has_v2())
+		ih << ", " << t.info_hash().v2;
```

Comment:
should `info_hash_t` just do this in its `operator<<`?

Reply:
The problem with defining `operator<<` for `info_hash_t` is that there is more than one thing it could do. Does it print one or both hashes as done here? Or only the "best" hash? Is the v2 hash truncated?